test/integration/wallet-web.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function getAddress() {
+  return 'rQDhz2ZNXmhxzCYwxU6qAbdxsHA4HV45Y2';
+}
+
+function getSecret() {
+  return 'shK6YXzwYfnFVn3YZSaMh5zuAddKx';
```

Comment:
We should avoid putting secrets in the source code so that people don't drain the funds and make us manually reset it a lot. You should be able to get it from the environment variables on CircleCI.